### ISSUE TITLE
A page named "Language": not able to edit modules

### ISSUE THREAD

**Issue by Timo-Breumelhof** [dnnsoftware/Dnn.Platform#5443]
## Description of bug
A page with the name "Language" is pretty much unusable as you cannot edit any module or edit the module settings.
In that case the page stays empty when you try to edit anything.

## Steps to reproduce
List the precise steps to reproduce the bug:
1. Create a page, name it "language" (can be a root or child page)
2. Place the HTML module on the page
3. Try to edit the module content or module settings
4. The edit page is "empty"

## Current behavior
See above

## Expected behavior
One should be able to create a page named "language" and use it or DNN should not allow the creation of a page with that name..

## Additional context
I do realize there's a chance this will not get fixed, but I still want to log this so someone with the same issue might find this issue and save him/herself the time I sent on this ;-)

## Affected version
* [ ] 10.00.00 alpha build
* [ ] 09.11.00 release candidate
* [x] 09.10.02 release candidate
* [x] 09.10.01 latest supported release

**Comment by jeremy-farrance** [comment]
I can't find it at the moment, but this sounds like a reserved word. I remember having similar issues a few years ago when the client wanted their page named, "Providers" and even when we forced it, we had problems with modules and other on page functionality. If I remember and find some reference to an issue or the discussion, I will post it. But also, it might NOT be related to this. :)

**Comment by Timo-Breumelhof** [comment]
Yes, we have also has a similar issue with a page named "login" which was used to redirect a user to an external login, while DNN was the "public site". Even with a custom login page that does not work.
IMO it would be good if DNN would not allow the creation of pages with there names

**Comment by bdukes** [comment]
#1247 added a check for _Login_ and _Register_ in DNN 8.0.1. (This check is called before `AddTab` in the Persona Bar and a couple of other places, so depending on how the page is created, it is possible to bypass it).

https://github.com/dnnsoftware/Dnn.Platform/blob/a2a0c6ec7e7cd2cd1dc32b9a6e118569f840e529/DNN%20Platform/Library/Entities/Tabs/TabController.cs#L50-L51

At the same time (DNN 8.0.1), #1342 also added a check the ensure that the page didn't match the _Do Not Rewrite URL Regular Expression_, which includes `/Providers/` (I wonder whether the slashes mean that it doesn't actually match…).

https://github.com/dnnsoftware/Dnn.Platform/blob/a2a0c6ec7e7cd2cd1dc32b9a6e118569f840e529/DNN%20Platform/Library/Entities/Tabs/TabController.cs#L1017-L1028

If it makes sense, we can add _Language_ to the list of unavailable page names. It would be better if we could figure out a way to have that page not broken, but it may not be a clean fix.

**Comment by Timo-Breumelhof** [comment]
@bdukes ok sounds good, but is "language" still used as a QS parameter or is that legacy?

**Comment by bdukes** [comment]
I think the typical path creates language-specific portal aliases, instead of the `language` QS parameter, but it's possible to use the  QS parameter
